mappings/net/minecraft/world/WorldProperties.mapping

```diff
@@ -123,8 +123,8 @@ CLASS none/bar net/minecraft/world/WorldProperties
 		ARG 0 damage
 	METHOD f setThunderTime (I)V
 		ARG 0 thunderTime
-	METHOD f setMapFeatures (Z)V
-		ARG 0 mapFeatures
+	METHOD f setStructures (Z)V
```

Comment:
I had gone over this one in IRC before mapping, should it be left as is or changed?